src/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
next would NOT let me not have this file.

Another important thing was changing the module from "commonjs" to "esnext", otherwise the app would crash. So this file is important and module must stay esnext in here and commonjs in the parent file